pdr_backend/pdr_dashboard/callbacks/callbacks_feeds.py

```diff
@@ -0,0 +1,254 @@
+import dash
+from dash import Input, Output, State, callback_context
+from pdr_backend.pdr_dashboard.pages.feeds import FeedsPage
+from pdr_backend.pdr_dashboard.dash_components.plots import (
+    get_feed_figures,
+    FeedModalFigures,
+)
+from pdr_backend.cli.arg_feeds import ArgFeed
+from pdr_backend.pdr_dashboard.util.format import format_table
+from pdr_backend.pdr_dashboard.util.data import get_feed_column_ids
+
+
+def filter_table_by_range(min_val, max_val, label_text):
+    ctx = callback_context
+    button_id = ctx.triggered[0]["prop_id"].split(".")[0]
+    if (not min_val and not max_val) or button_id == "clear_filters_button":
+        return label_text
+
+    return f"{label_text} {min_val if min_val else ''}-{max_val if max_val else ''}"
+
+
+def filter_condition(item, field, values):
+    return not values or item[field] in values
+
+
+def search_condition(item, search_value):
```

Comment:
same for this function. The content also suffers from bad spacing :) Leave some empty lines for clarity.